sdk/RELEASE.md

```diff
@@ -21,14 +23,6 @@
 
 * Support container environment variable in v2. [\#6515](https://github.com/kubeflow/pipelines/pull/6515)
 
-## Breaking Changes
```

Comment:
Maybe keep these empty sections? That seems how TFX handles the release note.
I don't have a strong opinion though. Up to you.

Reply:
I see, and I will add a N/A to it